jdaviz/configs/cubeviz/plugins/tests/test_parsers.py

```diff
@@ -219,11 +245,14 @@ def test_manga_cube(cubeviz_helper):
     uc.spectral_y_type = "Surface Brightness"
 
     se = cubeviz_helper.plugins['Spectral Extraction']
-    se.function = "Mean"
+    se.function = function
     se.extract()
-    extracted_max = cubeviz_helper.get_data("Spectrum (mean)").max()
-    assert_allclose(extracted_max.value, 2.836957E-18, rtol=1E-5)
```

Comment:
given that the tolerance was larger before I suggested a bigger one, I don't feel bad bumping it up.

Reply:
But I'll just bump it back up on 4.0.1 since there are no objections.